t/t0052-simple-ipc.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+test_description='simple command server'
+
+. ./test-lib.sh
+
+test_lazy_prereq SUPPORTS_SIMPLE_IPC '
+	test-tool simple-ipc SUPPORTS_SIMPLE_IPC
+'
+
+test_expect_success SUPPORTS_SIMPLE_IPC 'simple command server' '
+	{ test-tool simple-ipc daemon & } &&
```

Comment:
Should we track the process id and kill it when the test is over?

Reply:
Good idea!

Reply:
Done (finally!): https://github.com/microsoft/git/blob/45a31e825ce8a1c00a506db4b18a0fb9d4d8e3f7/t/t0052-simple-ipc.sh#L12-L23